oauth_provider/tests/auth.py

```diff
@@ -122,13 +126,14 @@ def _access_token(self, method=METHOD_URL_QUERY, **parameters_overriden):
         elif method==METHOD_URL_QUERY:
             response = self.c.get("/oauth/access_token/", parameters)
         elif method==METHOD_POST_REQUEST_BODY:
-            body = urllib.urlencode(parameters)
+            body = six.moves.urllib.parse.urlencode(parameters)
             response = self.c.post("/oauth/access_token/", body, content_type="application/x-www-form-urlencoded")
         else:
             raise NotImplementedError
 
         self.assertEqual(response.status_code, 200)
-        response_params = parse_qs(response.content)
+        response_params = parse_qs(response.content.decode('utf-8'))
```

Comment:
Response was coming in bytes.

Reply:
This view is converting data into bytes.
https://github.com/edx/django-oauth-plus/blob/awais786/BOM-67/oauth_provider/views.py#L217